diff --git a/node_modules/babel-plugin-transform-postcss/dist/plugin.js b/node_modules/babel-plugin-transform-postcss/dist/plugin.js
index 4523881..ecb5a28 100644
--- a/node_modules/babel-plugin-transform-postcss/dist/plugin.js
+++ b/node_modules/babel-plugin-transform-postcss/dist/plugin.js
@@ -31,7 +31,7 @@ const startServer = () => {
     stdio: 'inherit'
   });
 
-  server.unref();
+  // server.unref();
 };
 
 const stopServer = () => {
diff --git a/node_modules/babel-plugin-transform-postcss/dist/postcss-server.js b/node_modules/babel-plugin-transform-postcss/dist/postcss-server.js
index 4767d26..5daa4ed 100644
--- a/node_modules/babel-plugin-transform-postcss/dist/postcss-server.js
+++ b/node_modules/babel-plugin-transform-postcss/dist/postcss-server.js
@@ -148,7 +148,7 @@ const main = (() => {
       });
       server.on('listening', function () {
         const handler = function () {
-          _fs2.default.unlinkSync(socketPath); // eslint-disable-line no-sync
+          // _fs2.default.unlinkSync(socketPath); // eslint-disable-line no-sync
         };
 
         server.on('close', function () {
