diff --git a/node_modules/rehype-local-image-to-cloudinary/index.js b/node_modules/rehype-local-image-to-cloudinary/index.js
index 6a72153..56a1bf4 100644
--- a/node_modules/rehype-local-image-to-cloudinary/index.js
+++ b/node_modules/rehype-local-image-to-cloudinary/index.js
@@ -30,18 +30,20 @@ module.exports = ({
       });
 
       // add a srcSet for better perf on smaller viewports
-      node.properties.srcSet = [1800, 1200, 600, 300]
-        .map((width) => {
-          const srcSetTransformations = `${transformations},w_${width}`;
-          const url = getImageUrl({
-            fileName: cloudinaryName,
-            transformations: srcSetTransformations,
-            uploadFolder,
-          });
-
-          return `${url} ${width}w`;
-        })
-        .join();
+      if (path.extname(imagePath) !== '.gif') {
+        node.properties.srcSet = [1800, 1200, 600, 300]
+          .map((width) => {
+            const srcSetTransformations = `${transformations},w_${width}`;
+            const url = getImageUrl({
+              fileName: cloudinaryName,
+              transformations: srcSetTransformations,
+              uploadFolder,
+            });
+
+            return `${url} ${width}w`;
+          })
+          .join();
+      }
 
       // for browsers that support native lazy loading, add it
       node.properties.loading = 'lazy';
